bot/extensions/wikipedia_cog.py

```diff
@@ -0,0 +1,85 @@
+from discord.ext.commands import Cog, hybrid_command
+import discord
+import urllib.request
+import json
+
+
+def search_results(search):
```

Comment:
By convention, PEP8 recommends docstrings comments under all public modules, functions, classes, and methods. As mentioned in the PEP8 doc "_Docstrings are not necessary for non-public methods, but you should have a comment that describes what the method does. This comment should appear after the def line._". You can read more about this [here](https://peps.python.org/pep-0008/#documentation-strings). It is good practice/documentation to always do it :)